exercise-26.js

```diff
@@ -0,0 +1,56 @@
+function rent_prices() {
+    let average_rent = 0;
+    let apts_registered = [];
+    let number_input = 0;
+    let sum = 0;
+
+    function collect_apt_prices() {
+        const prompt = require("prompt-sync")({ sigint: true });
```

Comment:
Try moving this outside of the functions, people tend to use these are global imports.